lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1290,3 +1303,21 @@ def _process_marketplace_image_plan(
                 publisher=image_info.plan.publisher,
             )
         return plan
+
+    def _get_sriov_capability(
```

Comment:
It looks this method is not necessary. If it's really necessary, it should be put in the feature, make the sriov can be queried, not in platform.